integration-cli/docker_cli_network_unix_test.go

```diff
@@ -257,6 +257,31 @@ func (s *DockerNetworkSuite) TestDockerNetworkLsDefault(c *check.C) {
 	}
 }
 
+func (s *DockerNetworkSuite) TestDockerNetworkLsWithNoPredefined(c *check.C) {
```

Comment:
we should add a test for the type `builtin`, checking that custom networks don't show up.